listings/ch00-introduction/events/src/counter.cairo

```diff
@@ -0,0 +1,25 @@
+#[contract]
+mod SimpleCounter {
+    struct Storage {
+        // Counter value
+        _counter: u256,
+    }
+
+    #[event]
+    // Increment event  -  emitted when the counter is incremented.
+    fn Increment(counterVal: u256) {}
+
+    #[constructor]
+    fn constructor() {}
+
+    #[external]
+    fn increment() {
+        let mut counter: u256 = _counter::read();
+        // Increment counter
```

Comment:
I think this comment and the next one can be avoided, as the lines are already self-explanatory. Only keep the one related to the event